lib/tokenizers.js

```diff
@@ -37,19 +37,25 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
   }]);
 });
 
-export async function get({id} = {}) {
+export async function get({id, explain = false} = {}) {
   if(!(id && typeof id === 'string')) {
     throw new TypeError('"id" must be a string.');
   }
 
+  const query = {'tokenizer.id': id};
+  const projection = {_id: 0};
+  const collection = database.collections['tokenizer-tokenizer'];
+
+  if(explain) {
+    const cursor = await collection.find(query, projection);
```

Comment:
A comment here explaining the need for `collection.find` instead of `collection.findOne` in order to obtain the query explanation would be helpful. It will act as a useful reminder for our future selves that query explanations can only be executed on a subset of mongodb queries and that it requires cursors. https://docs.mongodb.com/upcoming/reference/method/db.collection.explain/